layouts/footer.slim

```diff
@@ -1,6 +1,35 @@
-footer.ui.center.aligned.container lang='en'
-  hr
-  p Copyright © 2016 by PPType, Inc.
+.ui.footer.black.inverted.segment lang='en'
```

Comment:
Prefer a `.ui.vertical.segment` since normal `.ui.segment` has a small `border-radius`. 

Compare:

![image](https://cloud.githubusercontent.com/assets/1164623/22060698/c8aeee0e-ddac-11e6-9d57-b4a9b0bb16c3.png)

with:

![image](https://cloud.githubusercontent.com/assets/1164623/22060714/e1c3f2f4-ddac-11e6-878e-3730c7358ddd.png)

And after you change to `.ui.vertical.segment`, please adjust `padding` or `margin` to make it balance.